src/components/Question.jsx

```diff
@@ -0,0 +1,48 @@
+import { nanoid } from "nanoid";
+import "../App.css";
+
+const Question = ({ q, id, handleClickAnswer }) => {
+  let result = q.answers;
+  const handleClick = (answer) => {
+    if (q.checked) {
+      return;
+    }
+    handleClickAnswer(id, answer);
+  };
```

Comment:
I would suggest using logical  and  operator to improve the performance and readability of your code.
```suggestion
  const handleClick = (answer) => q.checked && handleClickAnswer(id, answer);
```